From 8369a005eda8b5e45bdc5af422f90d1f045e7696 Mon Sep 17 00:00:00 2001
From: "zhiyuan.zhu" <zhiyuan.zhu@amlogic.com>
Date: Mon, 18 Nov 2019 15:14:01 +0800
Subject: [PATCH 29/29] frameworks-av: Join the code to the openlinux branch
 [4/5]

PD#SWPL-16484

Problem:
The DTVkit code is merged into the openlinux branch

Solution:
The DTVkit code is merged into the openlinux branch

Verify:
verify by newton

Change-Id: Id2e38361e1717f4700fcd7a25247aebe605bae91
Signed-off-by: zhiyuan.zhu <zhiyuan.zhu@amlogic.com>
---
 .../foundation/include/media/stagefright/foundation/MediaDefs.h  | 1 +
 1 file changed, 1 insertion(+)
 mode change 100644 => 100755 media/libstagefright/foundation/include/media/stagefright/foundation/MediaDefs.h

diff --git a/media/libstagefright/foundation/include/media/stagefright/foundation/MediaDefs.h b/media/libstagefright/foundation/include/media/stagefright/foundation/MediaDefs.h
old mode 100644
new mode 100755
index 25be89f4c..b165bcb94
--- a/media/libstagefright/foundation/include/media/stagefright/foundation/MediaDefs.h
+++ b/media/libstagefright/foundation/include/media/stagefright/foundation/MediaDefs.h
@@ -52,6 +52,7 @@ extern const char *MEDIA_MIMETYPE_AUDIO_AAC_ADTS;
 extern const char *MEDIA_MIMETYPE_AUDIO_MSGSM;
 extern const char *MEDIA_MIMETYPE_AUDIO_AC3;
 extern const char *MEDIA_MIMETYPE_AUDIO_EAC3;
+extern const char *MEDIA_MIMETYPE_AUDIO_AC4;
 extern const char *MEDIA_MIMETYPE_AUDIO_SCRAMBLED;
 
 extern const char *MEDIA_MIMETYPE_CONTAINER_MPEG4;
-- 
2.24.0

